src/yunohost/app.py

```diff
@@ -59,6 +59,7 @@
 APPS_CATALOG_CRON_PATH = "/etc/cron.daily/yunohost-fetch-apps-catalog"
 APPS_CATALOG_API_VERSION = 2
 APPS_CATALOG_DEFAULT_URL = "https://app.yunohost.org/default"
+APPS_DEFAULT_PHP_VERSION = "7.3"
```

Comment:
Is it really a good idea to set the default php version to `7.3` when we are on debian stretch. I thought that the default php version would be the official php version provided by debian so on stretch it's `7.0`.

I don't really mind this decision but the question is more to launch the debate about this because it still a important decision particularly sine we any many issues with sury.

Reply:
This PR will be merged into buster-unstable branch

Reply:
Ok in this case